docs/primitives/a-torus-knot.md

```diff
@@ -0,0 +1,38 @@
+---
+title: <a-torus-knot>
+type: primitives
+layout: docs
+parent_section: primitives
+---
+
+The torus knot primitive creates pretzel shapes using the [geometry][geometry]
```

Comment:
i'm intentionally using the long form name for components/primitives. sort of as a preference and to avoid codeblock noise